Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ApplyPaalmanPingsCorrection.py

```diff
@@ -7,7 +7,7 @@
 # pylint: disable=no-init,too-many-instance-attributes
 from __future__ import (absolute_import, division, print_function)
 
-import mantid.simpleapi as s_api
+from mantid.simpleapi import *
```

Comment:
```suggestion
from mantid.simpleapi import (AddSampleLog, ScaleX, ConvertUnits, RebinToWorkspace, CloneWorkspace, Divide)
```
Instead of importing all of simpleapi only import the algorithms you are going to use.

Reply:
Simpleapi is nothing but a collection of light wrapper functions, unlike kernel and api. I prefer importing all instead of very long list that could be growing much later on.